src/Code4Dependency.php

```
@@ -0,0 +1,164 @@
+use Ray\Di\NewInstance;
+
+use Ray\Di\SetterMethod;
+use Ray\Di\SetterMethods;
+use function implode;
+use function is_array;
+use function sprintf;
+
+use function var_export;
+use const PHP_EOL;
+
+class Code4Dependency extends Code
```

PHPStan (2.1.31)

Class Ray\Compiler\Code4Dependency extends final class Ray\Compiler\Code.


(class.extendsFinal)